Source/ImPlotWidgets/Public/LineFilledGraphWidget.h

```diff
@@ -4,14 +4,23 @@
 #include <Runtime/Core/Public/CoreMinimal.h>
 #include "ImPlotWidgets/Public/ImPlotWidget.h"
 #include <ImGui/Public/Widgets/ImGuiCustomWidget.h>
+#include <Runtime/Core/Public/Containers/ContainerAllocationPolicies.h>
+#include <Runtime/Core/Public/Math/UnitConversion.h>
+
 
 class IMPLOTWIDGETS_API SFilledGraphWidget : public SImPlotWidget
 {
 private:
 	using Super = SImPlotWidget;
 public:
 
-	void Construct(const FArguments& InArgs, const TArray<FImPlotData>& InData);
+	template<typename AllocatorType>
+	void Construct(const FArguments& InArgs, const TArray<FImPlotData, AllocatorType>& InData)
```

Comment:
I recommend taking this array by TArrayView instead `TArrayView<const FImPlotData>`